lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -2450,7 +2450,20 @@ removeDummyInput txBody =
                 ShelleyBasedEraShelley -> txBody
                 ShelleyBasedEraAllegra -> txBody
                 ShelleyBasedEraMary -> txBody
```

Comment:
@Unisay could we please revert 0fe8b4f8331f82e322c389c6dd27719d93938b1f and preserve the `bailOut` behaviour for Mary and earlier, rather than silently not removing the dummyInput?

Reply:
Done